content/en/security/cloud_security_management/agentless_scanning/_index.md

```diff
@@ -26,26 +26,25 @@ further_reading:
+|-------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Operating System                                | Linux                                                                                                                                          | Linux                                                                                                                                                                             |
+| Host Filesystem                                 | Btrfs, Ext2, Ext3, Ext4, xfs                                                                                                                   | Btrfs, Ext2, Ext3, Ext4, xfs                                                                                                                                                      |
+| Package Manager                                 | Deb (debian, ubuntu) <br> RPM (amazon-linux, fedora, redhat, centos) <br> APK (alpine)                                                         | Deb (debian, ubuntu) <br> RPM (fedora, redhat, centos) <br> APK (alpine)                                                                                                          |
+| Encryption                                      | AWS </br> Unencrypted </br> Encrypted - Platform Managed Key (PMK) </br> **Note**: Encrypted - Customer Managed Key (CMK) is **not** supported | Encrypted - Platform Managed Key (PMK): Azure Disk Storage Server-Side Encryption, Encryption at host </br> **Note**: Encrypted - Customer Managed Key (CMK) is **not** supported |
+| Container runtime                               | Docker, containerd </br> **Note**: CRI-O is **not** supported                                                                                  | Docker, containerd </br> **Note**: CRI-O is **not** supported                                                                                                                     |
+| Serverless                                      | AWS, AWS Lambda                                                                                                                                | [Request the feature][12]                                                                                                                                                         |
```

Comment:
Can you clarify whether Serverless support on Azure is available in Preview? This link goes to the preview sign up form. If it's not available, then I think we need to remove this link.

Reply:
Serverless on Azure is not supported by Agentless right now. The aim here was to add a request feature button to see how many potential customers would be interested.